frontend/src/app/home/home.page.scss

```diff
@@ -42,3 +42,22 @@ h3 {
 .logo-square {
   fill: var(--ion-color-secondary);
 }
+
+.custom-icon {
+  font-size: 64px;
+  color: lightgray;
```

Comment:
Please use Ionics color variables for this. I think "ion-color-light-shade" could be the one for this.

Also, this button is waaay to large :D  I'd suggest to set the font-size to 40px. On the button itself, you can also set the height to 40px (or maybe a bit more, I haven't tested that)

Reply:
ya ok. I saw somewhere that icon size should be multiples of 2^n so gave it 64 as 32 was too small. Will check with your suggestion.

Reply:
Hm I have never heard of this rule